doc/k8s/sqlflowhub/Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM sqlflow/sqlflow
 
+RUN echo "root:root" | chpasswd
```

Comment:
Maybe we don't need to create the user in Dockerfile, please take a look at the comment: https://github.com/sql-machine-learning/sqlflow/pull/1650#issuecomment-580991442 , thanks.